composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "3d4dd91bbd03dc469d055960c8260704",
+    "content-hash": "d4747f7667737667cee81d0e4fd5c4f9",
```

Comment:
Curious why there are composer.lock changes?

Reply:
I ran `composer update` to pull in the new block alignments plugin and other updates from when I was out. I believe anytime you run that or `composer install` the lock file gets updated.

Reply:
👍  Just want to confirm that you're running composer with PHP 8, not 8.1. (We've had a couple issues with php versions + composer conflicts.)